test/e2e/container_probe.go

```diff
@@ -75,7 +75,7 @@ var _ = Describe("Probing container", func() {
 		p, err := podClient.Create(makePodSpec(probe.withFailing().build(), nil))
 		expectNoError(err)
 
-		err = wait.Poll(poll, 90*time.Second, func() (bool, error) {
+		err = wait.Poll(poll, 120*time.Second, func() (bool, error) {
```

Comment:
Since this test needs to run the entire poll time to pass, maybe it would be better to add a preliminary phase that waits for the pod to be Running.